### PR TITLE
M3: Refactor Deprecations - MauticFocusBundle

### PR DESCRIPTION
**Pull request by kuzmany**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8012
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try create/edit focuss
3. Insert focus js in your page

### CODE REVIEW

**Comment by kuzmany** [comment]
@npracht can you re-test it?

**Comment by kuzmany** [comment]
@florentpetitjean Did you put script on your local page? 
Because your protocol is file:// 

image

**Comment by florentpetitjean** [comment]
I know I forgot something when I copied the script !
ok change it and go another error 
image
Tell me if I miss something else, not totally sure of what I do on this.

**Comment by escopecz** [comment]
@kuzmany I changed the goal branch to m2-to-m3 as that's where we test the M3 changes. Could you resolve the conflicts?

File: plugins/MauticFocusBundle/Form/Type/ContentType.php [hunk 1]
```diff
@@ -97,7 +100,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'font',
-            'choice',
+            ChoiceType::class,
```
Comment (escopecz) [inline plugins/MauticFocusBundle/Form/Type/ContentType.php]: All choices must have the values and labels switched. See more details in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit

**Comment by escopecz** [comment]
@kuzmany will you be able to finish this in a couple of days? We want to start testing on the 3.x branch which runs on Symfony 3.4 but all bundles must be refactored out of the deprecated code first. 

If not, we can take over. Just tell us so we don't duplicate the work.

**Comment by kuzmany** [comment]
@escopecz please review now

**Pull request merged**